### ISSUE TITLE
Update phet-lib usages to go through the phet namespace

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/papyrus#38]
Over slack @jonathanolson said 

> Moved over things to phet.X namespaces in built files, so we'll have phet.scenery.Node instead of scenery.Node. Details in https://github.com/phetsims/scenery/issues/966

The next time we make a new build of phet-lib we will probably need to update usages to go through the `phet` namespace instead of assuming that libs are on the window.

**Comment by jessegreenberg** [comment]
I would like to do this now so that I can use a new function in utterance-queue that is not in our build of phet-lib. We should also do this as soon as possible so there are fewer usages to update.

**Comment by jessegreenberg** [comment]
OK, phet-lib was updated in the above commit.  phet-lib components in papyrus are now accessed through the phet namespace. Then, I went through program code and updated references to go through the phet namespace as well. I tried to test most programs but the change was manual and I would not be surprised if this caused bugs.

I made a post about this in slack. We can fix references as problems come up, closing.
